input/fsh/Examples.fsh

```diff
@@ -347,3 +347,29 @@ Description: "mCODE Example for TNM Pathological Regional Nodes Category"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * valueCodeableConcept = AJCC#pN3 "N3"
+
+Instance: mCODECancerRelatedMedicationRequestExample01
+InstanceOf: CancerRelatedMedicationRequest
+Description: "mCODE Example for CancerRelatedMedicationRequest"
+* id = "mCODECancerRelatedMedicationRequestExample01"
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-cancer-related-medication-request"
+* subject = Reference(mCODEPatientExample01)
+* status = MedRequestStatus#active
+* intent = MedRequestIntent#order
```

Comment:
It must be a code system before the # sign.  (http://hl7.org/fhir/CodeSystem/medicationrequest-intent). Ditto for line 357

Reply:
right. that goes back to the alias change. It should be fine after that.  Interestingly, the QAReport didn't catch this.